cypress/e2e/buy-shirt.cy.ts

```diff
@@ -15,17 +16,26 @@ const loginPage = new LoginPage();
 const addressStepPage = new AddressStepPage();
 const shippingStepPage = new ShippingStepPage();
 const paymentStepPage = new PaymentStepPage();
+const shirtsListPage = new ShirtsListPage();
 
-describe("Buy a t-shirt", () => {
-  it("then should be bought a t-shirt", () => {
+describe('Buy a t-shirt', () => {
+  it('then should be bought a t-shirt', () => {
+    // Arrange
     const userData = {
-      emailUser: "aperdomobo@gmail.com",
-      emailPassword: "WorkshopProtractor",
+      emailUser: 'aperdomobo@gmail.com',
+      emailPassword: 'WorkshopProtractor'
     };
-
+    let tShirtList = [
+      '\n\t\t\t\t\t\t\tFaded Short Sleeve T-shirts\n\t\t\t\t\t\t'
+    ];
     menuContentPage.visitMenuContentPage();
+
+    // Act
     menuContentPage.goToTShirtMenu();
 
+    // Assert
+    shirtsListPage.validateItemsNumber(tShirtList.length);
+    shirtsListPage.validateItemsNames(tShirtList);
```

Comment:
we don't need this. What We expect in this file is to initialize the page objects in similar way than `cypress/e2e/dresses-list.cy.ts`